src/backend/weaviate/weaviate_manager.py

```diff
@@ -64,4 +64,55 @@ def add_weaviate_customer_class(self,customer_guid):
             return f"Error:{e}"
         except Exception as e:
             logger.error(f"Unexpected error:{e}")
-            return f"Unexpected error:{e}"
\ No newline at end of file
+            return f"Unexpected error:{e}"
+
+    def create_schema_per_customer_guid(self, customer_guid):
+        try:
+            schema = self.client.schema.get()
+            class_names = [cls["class"] for cls in schema.get("classes", [])]
+
+            class_name=self.generate_weaviate_class_name(customer_guid)
+
+            if class_name not in class_names:
+                schema_obj = {
+                    "class": class_name,
+                    "description": "Schema for storing semantic chunks of a document.",
```

Comment:
in description => "Schema for storing semantic chunks of a customer" +  customer_guid